examples/README.md

```diff
@@ -0,0 +1,177 @@
+# Create a new op in C++, compile it to .so library and load it in Python
+
+There are some operators in PyTorch library which are not supported in TRTorch. 
+To support these ops, users can register converters for missing ops. For example,
+if we try to compile a graph with a build of TRTorch that doesn't support the 
+[ELU](https://pytorch.org/docs/stable/generated/torch.nn.ELU.html) operation, 
+we will get following error:
+
+> Unable to convert node: %result.2 : Tensor = aten::elu(%x.1, %2, %3, %3) # /home/bowa/.local/lib/python3.6/site-packages/torch/nn/functional.py:1227:17 (conversion.AddLayer)
+Schema: aten::elu(Tensor self, Scalar alpha=1, Scalar scale=1, Scalar input_scale=1) -> (Tensor)
+Converter for aten::elu requested, but no such converter was found.
+If you need a converter for this operator, you can try implementing one yourself
+or request a converter: https://www.github.com/NVIDIA/TRTorch/issues
+
+## Writing Converter in C++
+We can register a converter for this operator in our application. You can find more 
+information on all the details of writing converters in the contributors documentation
+([Writing Converters](https://nvidia.github.io/TRTorch/contributors/writing_converters.html)).
+Once we are clear about these rules and writing patterns, we can create a seperate new C++ source file as:
+
+```c++
+#include "core/conversion/converters/converters.h"
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace conversion {
+namespace converters {
+namespace impl {
+namespace {
+
+auto acthardtanh TRTORCH_UNUSED = RegisterNodeConversionPatterns().pattern(

```

Comment:
probably use something like `auto actelu` here, you probably shouldnt use trtorch macros in this custom extention